src/utils/themes-manager.js

```diff
@@ -49,6 +51,34 @@ export class ThemesManager {
     }
   }
 
+  /**
+   * Makes color value for text.
+   * @param {string} inputColor  - css color in HEX format
+   * @returns {string} hex  - css color in HEX format
+   */
+  makeTextContrastColor(inputColor: string) {
+    const hsl = hexToHsl(inputColor);
```

Comment:
what is this calculation based on ?

Reply:
calculation taken from https://codepen.io/hillelkaltura/pen/NWOXMWM?editors=0110
Dana approved such calculation as part of Studio and KVE projects